README.md

```diff
@@ -39,6 +39,16 @@ yarn generate
 ```
 Newly generated type definitions will be available in `./generated`.
 
+### Configure typescript
+
+You should set 
+
+```json
+"esModuleInterop": true
+```
+
+in your tsconfig.json
```

Comment:
Thanks for the PR! Could you specify that this is only necessary if you would like to use default imports. The `* as` syntax can be used without setting `esModuleInterop` to `true`.

e.g. `import * as createLnRpc from '@radar/lnrpc';`